hasher-matcher-actioner/terraform/actions/main.tf

```diff
@@ -96,8 +96,9 @@ resource "aws_lambda_function" "action_evaluator" {
 
   environment {
     variables = {
-      ACTIONS_QUEUE_URL = aws_sqs_queue.actions_queue.id,
+      ACTIONS_QUEUE_URL   = aws_sqs_queue.actions_queue.id,
       REACTIONS_QUEUE_URL = aws_sqs_queue.reactions_queue.id,
+      HMA_CONFIG_TABLE    = var.hma_config_table.name
```

Comment:
watch for my code coming in with this change but named a different way. here and further on.